### ISSUE TITLE
Overlapping Textboxes

### ISSUE THREAD

**Issue by Meerkov** [SethRobinson/UGT#34]
![125713506-8e2d64e3-21b3-4c70-898d-5f79c268d051](https://user-images.githubusercontent.com/8139496/126403809-df90cfbc-e525-467e-817e-050c7f8bb204.png)

Sometimes, the vision API returns stray characters (like the giant I in this image). In this case, what is happening is that it detected `...` as `い`. And it thinks it's a separate textbox.

There are a few possible answers... One would be to try to merge the text. That could be good, but I'm not certain how to know where to insert the text (this is at the end, but it's possible it could be detected in the middle?).

The other idea is to throw away any text bounding box that is fully enclosed within another. This has some obvious benefits as perhaps the most annoying thing is when you have two boxes of text overlapping, because almost always the box that is underneath is the larger, more important box.

What do you think?

**Comment by SethRobinson** [comment]
Yeah, that would probably be a good idea.  Maybe I can try that next time I'm using something that has those issues...
